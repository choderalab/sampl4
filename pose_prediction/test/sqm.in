Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0002,
  tight_p_conv=1, scfconv=1.d-10,   qmcharge=0,
 /
   6    C1       -0.6740       -1.4090       -1.4800 
   6    C2        0.2170       -0.4690       -0.9560 
   6    C3        1.6760       -0.7190       -2.8730 
   6    C4        0.7920       -1.6710       -3.4250 
   6    C5       -0.3680       -1.9990       -2.7160 
   6    C6        1.3780       -0.1270       -1.6420 
   6    C7       -0.0590       -3.1480       -4.8250 
   6    C8        2.9290       -0.3420       -3.6030 
   7    N1        0.9550       -2.3380       -4.6300 
   7    N2       -0.1990       -3.9480       -5.9160 
  16    S1       -1.2600       -3.1760       -3.6070 
  17   Cl1        2.4320        1.0460       -0.9340 
   1    H1       -1.5780       -1.6710       -0.9380 
   1    H2       -0.0160       -0.0090        0.0020 
   1    H3        2.7350        0.5040       -4.2690 
   1    H4        3.7220       -0.0640       -2.9000 
   1    H5        3.3040       -1.1820       -4.1970 
   1    H6        0.4750       -3.9280       -6.6890 
   1    H7       -1.0310       -4.5330       -6.0510 

