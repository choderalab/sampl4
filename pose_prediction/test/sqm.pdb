ATOM      1  C1  MOL     1      -0.680  -1.411  -1.479
ATOM      2  C2  MOL     1       0.184  -0.452  -0.977
ATOM      3  C3  MOL     1       1.691  -0.703  -2.901
ATOM      4  C4  MOL     1       0.805  -1.680  -3.417
ATOM      5  C5  MOL     1      -0.381  -2.030  -2.691
ATOM      6  C6  MOL     1       1.351  -0.108  -1.683
ATOM      7  C7  MOL     1      -0.036  -3.223  -4.808
ATOM      8  C8  MOL     1       2.928  -0.321  -3.619
ATOM      9  N1  MOL     1       0.972  -2.362  -4.629
ATOM     10  N2  MOL     1      -0.146  -4.152  -5.836
ATOM     11  S1  MOL     1      -1.246  -3.224  -3.518
ATOM     12  Cl1 MOL     1       2.365   1.080  -1.000
ATOM     13  H1  MOL     1      -1.592  -1.682  -0.926
ATOM     14  H2  MOL     1      -0.041   0.044  -0.021
ATOM     15  H3  MOL     1       2.895   0.767  -3.881
ATOM     16  H4  MOL     1       3.818  -0.494  -2.961
ATOM     17  H5  MOL     1       3.071  -0.904  -4.560
ATOM     18  H6  MOL     1       0.543  -4.075  -6.556
ATOM     19  H7  MOL     1      -1.074  -4.252  -6.185
