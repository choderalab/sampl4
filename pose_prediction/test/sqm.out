            --------------------------------------------------------
                             AMBER SQM VERSION 13
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 31

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: S : M.J.S.DEWAR et al. INORG. CHEM., 29, 3881, (1990)
| QMMM: Cl: M.J.S.DEWAR et al. THEOCHEM, 180, 1, (1988)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.1D-09
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is disabled.
| QMMM: Using internal diagonalization routine (diag_routine=1).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C       -0.6740   -1.4090   -1.4800
  QMMM:     2        2      C        0.2170   -0.4690   -0.9560
  QMMM:     3        3      C        1.6760   -0.7190   -2.8730
  QMMM:     4        4      C        0.7920   -1.6710   -3.4250
  QMMM:     5        5      C       -0.3680   -1.9990   -2.7160
  QMMM:     6        6      C        1.3780   -0.1270   -1.6420
  QMMM:     7        7      C       -0.0590   -3.1480   -4.8250
  QMMM:     8        8      C        2.9290   -0.3420   -3.6030
  QMMM:     9        9      N        0.9550   -2.3380   -4.6300
  QMMM:    10       10      N       -0.1990   -3.9480   -5.9160
  QMMM:    11       11      S       -1.2600   -3.1760   -3.6070
  QMMM:    12       12      Cl       2.4320    1.0460   -0.9340
  QMMM:    13       13      H       -1.5780   -1.6710   -0.9380
  QMMM:    14       14      H       -0.0160   -0.0090    0.0020
  QMMM:    15       15      H        2.7350    0.5040   -4.2690
  QMMM:    16       16      H        3.7220   -0.0640   -2.9000
  QMMM:    17       17      H        3.3040   -1.1820   -4.1970
  QMMM:    18       18      H        0.4750   -3.9280   -6.6890
  QMMM:    19       19      H       -1.0310   -4.5330   -6.0510

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10       40.6408 kcal/mol        0.5455 kcal/(mol*A)
xmin    20       40.6196 kcal/mol        0.0242 kcal/(mol*A)
xmin    30       40.5999 kcal/mol        0.0612 kcal/(mol*A)
xmin    40       40.5745 kcal/mol        0.0166 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -40.5491  -38.2178  -37.1345  -34.5870  -33.7418  -29.8028  -27.4352  -25.1804
 -22.6409  -22.3931  -19.3242  -18.1261  -17.7344  -16.9241  -16.0386  -15.6869
 -15.5056  -14.5739  -14.3648  -13.8801  -13.3651  -13.2185  -12.6862  -12.5702
 -12.1929  -11.7013  -11.1883  -11.0374  -10.7090   -8.9266   -8.8386   -0.5064
  -0.0687    0.0146    0.6493    1.1463    1.3613    2.3634    2.4539    3.4210
   3.6268    3.7411    4.0889    4.1641    4.3960    4.6216    4.6820    4.8905
   5.0279    5.0818    5.1793    5.2771    5.3774    5.6807    6.2907

 Heat of formation   =         40.57435512 kcal/mol  (        1.75943607 eV)

 Total SCF energy    =     -48481.65891239 kcal/mol  (    -2102.32248872 eV)
 Electronic energy   =    -234056.06178471 kcal/mol  (   -10149.43245240 eV)
 Core-core repulsion =     185574.40287231 kcal/mol  (     8047.10996368 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      C                 -0.082
     2      C                 -0.124
     3      C                 -0.012
     4      C                 -0.015
     5      C                 -0.323
     6      C                 -0.063
     7      C                 -0.098
     8      C                 -0.181
     9      N                 -0.168
    10      N                 -0.298
    11      S                  0.386
    12      Cl                -0.015
    13      H                  0.143
    14      H                  0.150
    15      H                  0.091
    16      H                  0.092
    17      H                  0.108
    18      H                  0.208
    19      H                  0.200
 Total Mulliken Charge =      -0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE    -2.094   -0.750   -1.378    2.617
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C       -0.6802   -1.4113   -1.4789
  QMMM:     2        2      C        0.1842   -0.4521   -0.9774
  QMMM:     3        3      C        1.6908   -0.7027   -2.9013
  QMMM:     4        4      C        0.8055   -1.6801   -3.4171
  QMMM:     5        5      C       -0.3806   -2.0304   -2.6909
  QMMM:     6        6      C        1.3511   -0.1079   -1.6830
  QMMM:     7        7      C       -0.0362   -3.2234   -4.8084
  QMMM:     8        8      C        2.9280   -0.3214   -3.6188
  QMMM:     9        9      N        0.9724   -2.3625   -4.6288
  QMMM:    10       10      N       -0.1456   -4.1515   -5.8361
  QMMM:    11       11      S       -1.2456   -3.2236   -3.5180
  QMMM:    12       12      Cl       2.3645    1.0801   -0.9999
  QMMM:    13       13      H       -1.5916   -1.6822   -0.9256
  QMMM:    14       14      H       -0.0405    0.0440   -0.0205
  QMMM:    15       15      H        2.8952    0.7669   -3.8812
  QMMM:    16       16      H        3.8183   -0.4937   -2.9614
  QMMM:    17       17      H        3.0715   -0.9043   -4.5600
  QMMM:    18       18      H        0.5429   -4.0751   -6.5563
  QMMM:    19       19      H       -1.0743   -4.2517   -6.1853

           --------- Calculation Completed ----------

